src/recordlinker/schemas/pii.py

```diff
@@ -441,27 +438,25 @@ def feature_iter(self, feature: Feature) -> typing.Iterator[str]:
                     yield str(race)
         elif attribute == FeatureAttribute.TELECOM:
             for telecom in self.telecom:
-                if telecom.system is None:
-                    yield telecom.value.strip().lower()
-                    continue
-
-                handlers = Telecom.get_system_handlers()
-
-                if telecom.system in handlers:
-                    value = getattr(telecom, handlers[telecom.system])()
-                    if value:
-                        yield value
-
+                value = telecom.value
+                if telecom.system == "phone":
+                    # Use national number for comparison
+                    phone = normalize_text(str(phonenumbers.parse(telecom.value).national_number))
```

Comment:
```suggestion
                    phone = normalize_text(str(phonenumbers.parse(value).national_number))
```
Alternatively, the `value` local doesn't add much.  you could just change line 448 to `yield telecom.value`